model/jpa/src/main/java/org/keycloak/models/jpa/entities/UserConsentEntity.java

```diff
@@ -17,6 +17,8 @@
 
 package org.keycloak.models.jpa.entities;
 
+import org.keycloak.grant.jpa.UserGrantEntity;
+
```

Comment:
By recent keycloak's convention, it might be better not to add or modify JPA entity because it causes a schema change. For example, Client Policies/Profiles avoid adding entities by storing JSON string itself to realm's attribute (existing key-value store).

Reply:
will be remove and stored as described above